Source/Core/Core/Core.cpp

```diff
@@ -503,11 +503,12 @@ static void EmuThread(std::unique_ptr<BootParameters> boot, WindowSystemInfo wsi
   bool sync_sd_folder = core_parameter.bWii && Config::Get(Config::MAIN_WII_SD_CARD) &&
                         Config::Get(Config::MAIN_WII_SD_CARD_ENABLE_FOLDER_SYNC);
   if (sync_sd_folder)
-    sync_sd_folder = Common::SyncSDFolderToSDImage(Core::WantsDeterminism());
+    sync_sd_folder =
+        Common::SyncSDFolderToSDImage([]() { return false; }, Core::WantsDeterminism());
```

Comment:
We probably want curlies here. (But GitHub says no to me adding a suggestion, because I included the `if` in my selection)